blocks/landing-page-hero/landing-page-hero.js

```diff
@@ -103,14 +132,68 @@ function buildOverlay(overlayRawContent) {
   return overlayContent;
 }
 
-function decorateCtasContent(ctasContent, stylesList) {
+function getFormSection(button, formStyle = 'form-overlay-overlay') {
+  const formSection = section({ class: 'form-overlay' });
+  formSection.innerHTML = `<div class="form-overlay-trigger">
+      <a class="btn form-overlay-button">${button.innerText}</a>
+    </div>
+    <div class="${formStyle} " role="dialog" aria-modal="true" aria-label="form">
+      <!-Form Content Goes Here -->
+    </div>`;
+
+  return formSection;
+}
+
+async function fetchFormThanksElem(thanksUrl) {
+  const thanksElem = div({ class: 'vlt-form-success' });
+  const resp = await fetch(thanksUrl);
+  if (resp.ok) {
+    thanksElem.innerHTML = await resp.text();
+  } else {
+  // Come up with some default thanks text to display if no thanks HTML is found
```

Comment:
In the past, I've had a `Thank You` property on the block so this could be defined by the author.
You could change this to accept text or a URL.
Either way, the author should be able to define the thank you.

Reply:
Heya @rrusher!  Yeah that makes total sense, and was the way that @synox implemented the form submission content in the original `embedded-form` block.  I was trying to avoid making the landing-page-hero block configuration less cumbersome than it already is, but I think that's unfortunately unavoidable with such a complicated component.  

I would like to get this guy merged in before it bloats (and since I'm already working on another branch based off this one so I would like to any significant rewrites here), but I put in a new issue #163 to get this added in, which I will start working on once this guy is merged in and the PR for #159  is in.